app/models/product.rb

```diff
@@ -2,14 +2,16 @@ class Product < ApplicationRecord
   PERMALINK_REGEX = /[[:alnum:]]+/
```

Comment:
where is scope for all enabled products?

Reply:
Didn't create scope in last PR of Associations, I have created now after PR review, will merge the Associations branch after final review.